lua/cloak/init.lua

```diff
@@ -53,6 +54,14 @@ M.cloak = function(cloak_pattern)
     require('cmp').setup.buffer({ enabled = false })
   end
 
+  local function determine_pattern(first_line, last_line)
```

Comment:
Any reason you are calling this first_line and last_line, aren't these the first_column and last_column or first_char and last_char?

Line makes me think this is cloaking multiple lines, which isn't the case.

I would also call the function `determine_replacement` instead.

```suggestion
  local function determine_replacement(first_col, last_col)
```

Reply:
Sorry about that, I mistook the intention with the first and last stuff. 

Applying the feedback!